validate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Validate AppStream catalog.
```

Comment:
Nitpick: Would be useful to add a license and copyright and necessary SPDX runes for those.

(same for all the new files in this MR)

Reply:
Good call.